pyproject.toml

```diff
@@ -7,11 +7,31 @@ license = "MIT"
 
 [tool.poetry.dependencies]
 python = "^3.8,<3.11"
+
+[tool.poetry.group.dev.dependencies]
+demisto-sdk = {git = "https://github.com/demisto/demisto-sdk.git", rev = "1.7.6-mypy-hotfix"}
```

Comment:
please change this to sha1 (we are not using branches for sdk hot fixes)

Reply:
@esharf 
We agreed with @yaakovi to use the branch in to `pyproject.toml` since it's more readable, and the actual installation is from the `poetry.lock` file, where it saves the latest commit of this branch when the file was created.
If more changes will be pushed to this branch, `poetry` will still use the same commit, until the next `poetry lock`.